AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/editor_script_canvas.py

```diff
@@ -88,7 +88,7 @@ def get_component_graph_file(self) -> str:
 
     def set_variable_value(self, variable_name: str, variable_state: VariableState, variable_value) -> str:
         """
-        o3de/o3de#13344. We currently only support strings and boolean variables.
+        o3de/o3de#13344. We currently only support strings, entity.id and boolean variables.
         Function for setting a value to a variable on the script canvas component.
 
         Note: The variable must be exposed to the component. this can be done through the script canvas node inspector.
```

Comment:
Put the GHI below the description but above the note so that way the autocomplete will give the relevant information of the usage.

Reply:
reorganized this per request.